builds/cmake/Modules/FindOggVorbis.cmake

```diff
@@ -0,0 +1,10 @@
+find_path(OGGVORBIS_INCLUDE_DIR ogg/ogg.h)
+find_library(OGGVORBIS_LIBRARY NAMES ogg libogg)
```

Comment:
the required lib is vorbisfile (which provides the same high-level api as tremor), so this will not work correctly